src/cascadia/TerminalSettingsModel/VsSetupConfiguration.h

```diff
@@ -120,17 +120,15 @@ namespace Microsoft::Terminal::Settings::Model
         private:
             friend class VsSetupConfiguration;
 
-            VsSetupInstance(const ComPtrSetupQuery pQuery, const ComPtrSetupInstance pInstance) :
-                query(pQuery),
-                helper(pQuery.query<ISetupHelper>()),
-                inst(pInstance),
+            VsSetupInstance(ComPtrSetupQuery pQuery, ComPtrSetupInstance pInstance) :
+                query(std::move(pQuery)),

```

Comment:
Taken by copy, moved into storage. We could take the Instance as an rvalue ref, but I didn't see the value in doing that.